### ISSUE TITLE
Possible typo in population size of first regime

### ISSUE THREAD

**Issue by vinaym815** [CMA-ES/libcmaes#229]
https://github.com/CMA-ES/libcmaes/blob/bf994354c072e58f3125014a40029ee65cd7e00d/src/bipopcmastrategy.cc#L122

Should not the value sampled from uniform distribution be squared as suggested in:
https://hal.inria.fr/inria-00382093/document
https://hal.inria.fr/hal-00818596v1/document

**Comment by beniz** [comment]
![image](https://user-images.githubusercontent.com/3530657/133077931-cffdc518-f8a4-43e6-8373-c32f615cb724.png)

Hi @vinaym815 good catch, possibly, let's see whether @nikohansen can confirm ?

**Comment by vinaym815** [comment]
Also in the line above the one mentioned here, it might be desirable to change the experssion to that proposed in second paper.

sigma expression first publication
![sigma2](https://user-images.githubusercontent.com/35987596/133079718-ff95beca-f90b-42ad-a77b-df54d82b2c05.png)

sigma expression second publication
![sigma1](https://user-images.githubusercontent.com/35987596/133079715-31f939ed-0d37-4e7c-a0bc-378c0ba7ddb2.png)

**Comment by nikohansen** [comment]
Yes, I agree with both, 2 -> default initial sigma and the uniform random number should be squared first.
